client/service/src/lib.rs

```diff
@@ -340,7 +341,10 @@ impl<TBl: Unpin, TCl, TSc: Unpin, TNetStatus, TNet, TTxPool, TOc> Future for
 		}
 
 		while let Poll::Ready(Some((task_to_spawn, name))) = Pin::new(&mut this.to_spawn_rx).poll_next(cx) {
-			(this.task_executor)(Box::pin(futures_diagnose::diagnose(name, task_to_spawn)));
+			#[cfg(not(target_os = "unknown"))]
+			let task_to_spawn = futures_diagnose::diagnose(name, task_to_spawn);
```

Comment:
Is `futures_diagnose` not compiling for `wasm32-unknown`?

Reply:
It does compile, but it panics because of `std::time::Instant::now()`.

Reply:
Can we make it not panic?

Reply:
Yep, if we use `wasm-timer` in it.

Reply:
See https://github.com/tomaka/futures-diagnose/pull/4.